### DISCUSSION TITLE
Can I "turn off" built-in import css support to reduce bundle size?

### DISCUSSION THREAD

**Discussion by ardok** [vercel/next.js#18938] [Help] [answered]
I'm not sure how this works exactly, so please correct me if I'm wrong.

* Does this built-in import css support increase the final bundle size?
  * I mainly want to make sure that users who visit my site to use as less bandwidth as possible.
  * If yes, is there a way to turn it off somehow? I want to use a different styling framework.

**Comment by jamesmosier** [comment]
If you use Next's built-in CSS it will output the CSS to separate CSS files in production mode, therefore your JS bundle sizes are not effected. 

> I want to use a different styling framework

What styling framework are you wanting to use?

**Reply by ardok** [reply]
I want to use [Styletron](https://www.styletron.org/), or maybe something else that does css in js. Still unsure, but most likely Styletron.

**Comment by Timer** [comment, marked as answer]
The build-in CSS support has zero-overhead on your app if not in use. There's no need to "turn it off", just don't use it and your application size won't gain the bytes.
